.github/workflows/update-urls.yml

```
@@ -25,7 +25,7 @@ jobs:
           cache: 'maven'
       - name: Build and run url updater
         run: |
-          cd cli
+          cd url-updater
           mvn -B -ntp -Dstyle.color=always install
```

this is not going to work since `url-updater` depends on `cli` so if you only build `url-updater` without building `cli` it will fail.
This currently does not fail because the artifacts will be downloaded by maven from nexus.
But when someone (our release process) sets a new version in the POM and then this action is going to be executed, it will fail because `cli` artifact is missing.
```suggestion
          mvn -B -ntp -Dstyle.color=always -pl url-updater -am install
```

---

Adjusted.